ios/fastlane/Fastfile

```diff
@@ -37,7 +37,7 @@ def get_changelog()
 end
 
 
-def get_semver_from_branch()
+def get_semver_from_branch(with_meta=false)
 
   # official semver regex from https://semver.org/, slightly modified for branch names
```

Comment:
Maybe the comment could explain what was modified to understand the difference. :)